### PR TITLE
Verify that attach file in topic is removed

### PR DESCRIPTION
**Pull request by esmaoui**

### CODE REVIEW

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java [hunk 1]
```diff
@@ -891,4 +891,7 @@
   public final By                     ELEMENT_MOVE_TOPIC_POPUP                           = By.id("UIForumPopupWindow");
 
   public static final SelenideElement ELEMENT_BUTTON_SUBMIT_POST= $(byXpath("//*[@id=\"UIPostForm\"]/div[3]/button[1]"));
+
+  public static final SelenideElement ELEMENT_REMOVE_ICON= $(byClassName("uiIconDelete "));
```
Comment (ilyeskallala) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java]: useless locator

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopic.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        forumTopicManagement.startTopic(topic, topic, "", "");
+        info("Go to a topic");
+        forumHomePage.goToTopic(topic);
+        forumTopicManagement.selectItemMoreActionMenuTopic(ForumTopicManagement.specifMoreActionMenuTopic.EDIT);
+        $(byId("UIEditTopicContainer")).find(byClassName("uiIconDelete")).click();
+        $(ELEMENT_SUBMIT_BUTTON).click();
+        info("verify that the attach file is removed");
+        forumHomePage.goToTopic(topic);
+        forumTopicManagement.selectItemMoreActionMenuTopic(ForumTopicManagement.specifMoreActionMenuTopic.EDIT);
+        $(byId("UIEditTopicContainer")).find(byClassName("uiIconDelete")).shouldNotBe(Condition.visible);
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopic.java]: you should declare this element in ForumLocator.java

File: commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java [hunk 3]
```diff
@@ -82,10 +82,10 @@ public void afterEach() {
     Screenshots.finishContext();
   }
 
-  @AfterEach
+  /*@AfterEach
```
Comment (ilyeskallala) [inline commons/src/main/java/org/exoplatform/platform/qa/ui/commons/Base.java]: should not comment this code

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java [hunk 4]
```diff
@@ -886,12 +886,9 @@
   public static final By              ELEMENT_BUTTON_DELETE_MESSAGE                      = byClassName("uiIconDelete");
   public static final SelenideElement ELEMENT_UPLOAD_CATEGORIE_FORUM                     =
                                                                      $(byId("FileUpload")).find(byClassName("file"));
-  public final String                 ELEMENT_EDIT_POST                                  =
-                                                        "//*[@class='postViewTitle' and contains(text(),'{$title}')]/../..//*[@data-original-title='Edit This Post']";
-  public final By                     ELEMENT_MOVE_TOPIC_POPUP                           = By.id("UIForumPopupWindow");
```
Comment (ilyeskallala) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java]: why did you remove these locators ?

Comment (esmaoui) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java]: Useless locators

Comment (aboughdiri) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/forum/ForumLocator.java]: corrected

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopic.java (hunk 2 above)
Comment (ilyeskallala) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopic.java]: comment not considered

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopicTestIT.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+   * category, forum, topic Step Description: - Create a category - Create a forum
+   * - Create a topic Input Data: Expected Outcome: Category, forum, topic are
+   * created successfully Step number: 2 Step Name: Edit topic and remove attach file:
+   * Expected Outcome: the attach file is removed successfully
+   */
+
+
+    @Tag("FORUM-1382")
+    @Test
+    public void test_VerifyRemoveAttachFile(){
```
Comment (ilyeskallala) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/RemoveAttachFileInTopicTestIT.java]: move this TC to ForumTopicTestIT class
